### PR TITLE
libbpg 0.9.6

### PR DESCRIPTION
**Pull request by Nyx0uf**

### CODE REVIEW

File: Library/Formula/libbpg.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libbpg < Formula
   desc "Image format meant to improve on JPEG quality and file size"
   homepage "http://bellard.org/bpg/"
-  url "http://bellard.org/bpg/libbpg-0.9.5.tar.gz"
-  sha256 "30de1d0099920e24b7c9aae4d4e6b62f446823f0a1d52eb195dfc25c662ee203"
+  url "http://bellard.org/bpg/libbpg-0.9.6.tar.gz"
+  sha256 "931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1"
```
Comment (mikemcquaid) [inline Library/Formula/libbpg.rb]: This seems wrong? http://bot.brew.sh/job/Homebrew%20Pull%20Requests/37223/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/fetch_libbpg/

Comment (Nyx0uf) [inline Library/Formula/libbpg.rb]: that's strange, after I download the file I get `931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1` as sha256 checksum.

With both `shasum -a 256 file` and `openssl sha256 file`

Comment (mikemcquaid) [inline Library/Formula/libbpg.rb]: Not what the bot is getting.

Comment (rakkesh) [inline Library/Formula/libbpg.rb]: @Nyx0uf Did you download `http://bellard.org/bpg/libbpg-0.9.6.tar.gz` using Safari? Safari  decompresses a gzipped ( or any compressed ) file. 

```
$ shasum -a 256 libbpg-0.9.6.tar
931abf2744922973e2d57d62b91ef3dd033f0d5d1d829e8f04473f21531311d1  libbpg-0.9.6.tar
```

Comment (Nyx0uf) [inline Library/Formula/libbpg.rb]: No I'm using Chrome, but it looks like it does the same thing as Safari. Just tried to download with wget and I'm getting the same sha256 as the bot.

**Comment by rakkesh** [comment]
The build process now seems to depend on `cmake`.

**Comment by mikemcquaid** [comment]
@rakkesh Could you open a new pull request? Thanks!

**Comment by rakkesh** [comment]
Created a PR here, https://github.com/Homebrew/homebrew/pull/46464.

**Pull request closed without merging**